### PR TITLE
add dlrm inf batch latency and migrate it to bmlogger

### PR DESCRIPTION
**Pull request by erichan1**
1. Migrate DLRM to bmlogger away from fb5logger
2. Add dlrm inference batch latency

Tested with:
gpurun ./run_dlrm_ootb_infer.sh -l results
gpurun ./run_dlrm_ootb_train.sh -l results

### CODE REVIEW

**Review by aaronenyeshi — CHANGES_REQUESTED** [review]
I've noticed that param directory is modified, and could be a mistake?

**Comment by erichan1** [comment]
> I've noticed that param directory is modified, and could be a mistake?

Yes that is a mistake. Thanks for catching this and being great about PR reviewing in general.

File: benchmarks/dlrm/ootb/dlrm_s_pytorch.py [hunk 1]
```diff
@@ -2361,8 +2368,8 @@ def run():
                                     },
                                 )
                             break
-                if k == 0 and args.fb5logger is not None:
-                    fb5logger.run_stop(nbatches - args.warmup_steps, args.mini_batch_size)
+                if k == 0:
```
Comment (aaronenyeshi) [inline benchmarks/dlrm/ootb/dlrm_s_pytorch.py:2371]: Are we only capturing the first epoch?

**Review by aaronenyeshi — APPROVED** [review]
LGTM! Just a small question to help me understand better.

File: benchmarks/dlrm/ootb/dlrm_s_pytorch.py (hunk 1 above)
Comment (erichan1) [reply benchmarks/dlrm/ootb/dlrm_s_pytorch.py:2371]: Looks like it. I did not write this personally but seems like we always run 1 epoch so it isn't an issue. In the run commands we do not set a value for args.nepochs and the default is 1. Might be better to change this to log for any number of epochs in another PR.

**Pull request merged**